Sources/PointFree/SVG.swift

```diff
@@ -197,3 +182,27 @@ let twitterIconSvgBase64 = base64EncodedString <| """
     </g>
 </svg>
 """
+
+let pointFreePointersLogoSvgBase64 = base64EncodedString <| """
```

Comment:
this is a gigantic SVG with the vectorized font for "Pointers" in the logo. weighs in at 48kb. I do believe it's possible for us to just reference the google font in SVG, but didn't have time to look into. will add it to my checklist.